user_guide/admin_capabilities.rst

```diff
@@ -0,0 +1,461 @@
+.. _admin-capabilities:
+
+Capabilities List
+=================
+
+The *"Capabilities"* make an important part of the access and permission settings defined in the system, as described in 
+the :ref:`Access and Permissions Basics guide <user-guide-user-management-permissions-basic>`. 
+
+Depending on their specific roles, users will have access to different functionalities. For example, some users may not 
+be able to assign tags, share grids, or create new users.
+
+For each role and capability pair there are only two options:
+
+For each role and capability pair there are only two options:
+
+- **None**: users with the role won’t be able to use the functionality.
+- ***System***: users with the role will be able to use the functionality for all the records created within their 
+  OroCRM instance they’ve logged in into.
```

Comment:
Edit: **System**: users with this role will be able to use the functionality for all the records created within the OroCRM instance they’ve logged into.